gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/QuantityMapper.java

```diff
@@ -0,0 +1,58 @@
+package uk.nhs.adaptors.pss.translator.mapper;
+
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.hl7.fhir.dstu3.model.Quantity;
+import org.hl7.v3.PQR;
+import org.hl7.v3.Value;
+
+public class QuantityMapper {
+    private static final String UNIT_SYSTEM = "http://unitsofmeasure.org";
+    private static final String TYPE_IVL_PQ = "IVL_PQ";
+
+
+    public Quantity mapQuantity(Value value) {
+        Quantity quantity = new Quantity();
+
+        if (TYPE_IVL_PQ.equals(value.getType())) {
+            if (value.getHigh() != null) {
```

Comment:
Maybe you could add a variable for `value.getHigh()` over this line, as it's used in a few places later